packages/salesforcedx-sobjects-faux-generator/src/describe/sObjectDescribe.ts

```diff
@@ -161,30 +162,76 @@ export enum SObjectCategory {
   CUSTOM = 'CUSTOM'
 }
 
+type SubRequest = { method: string; url: string };
+type BatchRequest = { batchRequests: SubRequest[] };
+
+type SubResponse = { statusCode: number; result: SObject };
+
+type BatchResponse = { hasErrors: boolean; results: SubResponse[] };
+
 export class SObjectDescribe {
+  private accessToken: string;
+  private instanceUrl: string;
+  // TODO should get the proper version from ??
+  private readonly servicesPath: string = 'services/data';
```

Comment:
If you use a readonly field, there is a slot created for SObjectDescribe instance. This seems redundant since all of them are the same.
Does it make more sense to pull this out of the class and use a `const` instead?

Reply:
You are right, it's a bit cleaner to pull it out.  I put it there because the other relevant strings were dynamic in some way tied to the api version which should really be found from the workspace.  Needs to be cleaned up properly once I have the code to obtain that info properly.  I"m going to just leave it until then.   On my list of things to figure out from force-com-toolbelt code.